sdk/identity/identity/src/msal/browserFlows/browserCommon.ts

```diff
@@ -73,6 +75,10 @@ export abstract class MsalBrowser extends MsalBaseUtilities implements MsalBrows
     super(options);
     this.logger = options.logger;
     this.loginStyle = options.loginStyle;
+    if (!options.clientId) {
+      throw new CredentialUnavailable("A client ID is required in browsers");
```

Comment:
This is fine but reminder we might change this `CredentialUnavailableError`

Reply:
I made an issue to track that: https://github.com/Azure/azure-sdk-for-js/issues/14662